packages/ember-testing/tests/acceptance_test.js

```diff
@@ -295,3 +302,37 @@ QUnit.test("only enters the index route once when visiting /", function() {
     equal(indexHitCount, 1, 'should hit index once when visiting /');
   });
 });
+
+test("test must not finish while asyncHelpers are pending", function () {
```

Comment:
We have updated to prepare for QUnit 2.0, and removed `test` from our `.jshintrc`.  Can you change this to `QUnit.test`?